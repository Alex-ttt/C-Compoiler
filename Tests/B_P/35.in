int main()
{
	int a = 3;
	float c = 5.5;
	a &= 1;
	a |= 1;
	a <<= 1;
	a >>= 1;
	a &= c;
	a |= c;
	a <<= c;
	a >>= c;
}