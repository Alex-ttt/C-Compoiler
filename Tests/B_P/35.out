Expression must have integral type	 line 9, col 59

